kie-server-parent/kie-server-services/kie-server-services-openshift/src/test/java/org/kie/server/services/openshift/impl/storage/cloud/KieServerStateOpenShiftRepositoryEnhancedTest.java

```diff
@@ -0,0 +1,24 @@
+package org.kie.server.services.openshift.impl.storage.cloud;
+
+import org.junit.Test;
+import org.kie.server.api.KieServerConstants;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+public class KieServerStateOpenShiftRepositoryEnhancedTest extends KieServerStateOpenShiftRepositoryTest {
```

Comment:
What is the reason for separating this test from other ones?

Reply:
The main reason is that the tests inside this test case class will interfere the execution of other tests due to the need to set System properties. 

https://github.com/rhtevan/droolsjbpm-integration/blob/041052b96b020cc64bd7bd84a9f09d56ec2defbc/kie-server-parent/kie-server-services/kie-server-services-openshift/src/test/java/org/kie/server/services/openshift/impl/storage/cloud/KieServerStateOpenShiftRepositoryEnhancedTest.java#L34